assets/icons/file_icons/file_types.json

```diff
@@ -9,34 +9,48 @@
     "aac": "audio",
     "accdb": "storage",
     "app.src": "erlang",
+    "ascx": "template",
     "astro": "astro",
+    "atom": "template",
     "avi": "video",
     "avif": "image",
+    "axaml": "template",
+    "axml": "template",
     "bak": "backup",
     "bash": "terminal",
     "bash_aliases": "terminal",
     "bash_logout": "terminal",
     "bash_profile": "terminal",
     "bashrc": "terminal",
     "bmp": "image",
+    "bpmn": "template",
     "c": "c",
     "cc": "cpp",
     "cjs": "javascript",
     "coffee": "coffeescript",
     "conf": "settings",
     "cpp": "cpp",
+    "csl": "template",
+    "csproj": "template",
+    "csproj.user": "template",
     "css": "css",
     "csv": "storage",
     "cts": "typescript",
     "dart": "dart",
     "dat": "storage",
     "db": "storage",
     "dbf": "storage",
+    "dita": "template",
+    "ditamap": "template",
     "dll": "storage",
     "doc": "document",
     "docx": "document",
+    "dtd": "template",
+    "dtml": "template",
     "eex": "elixir",
     "elm": "elm",
+    "ent": "template",
+    "entitlments": "template",
```

Comment:
Is this a typo? Should it be “entitlements”?

Reply:
Yes, I'll change it